src/main/java/de/chojo/repbot/data/wrapper/AbuseSettings.java

```diff
@@ -0,0 +1,71 @@
+package de.chojo.repbot.data.wrapper;
+
+import net.dv8tion.jda.api.entities.Message;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+public class AbuseSettings {
+    private int cooldown;
+    private int maxMessageAge;
+    private int minMessages;
+    private boolean donorContext;
+    private boolean receiverContext;
+
+    public AbuseSettings(int cooldown, int maxMessageAge, int minMessages, boolean donorContext, boolean receiverContext) {
+        this.cooldown = cooldown;
+        this.maxMessageAge = maxMessageAge;
+        this.minMessages = minMessages;
+        this.donorContext = donorContext;
+        this.receiverContext = receiverContext;
+    }
+
+    public AbuseSettings() {
+    }
+
```

Comment:
```suggestion
```

Reply:
this constructor should contain reasonable default values. Gonna change this.